hack/ci-e2e.sh

```diff
@@ -17,6 +17,18 @@ cd "${REPO_ROOT}" || exit 1
 # BMO_E2E_EMULATOR can be set to either "vbmc" or "sushy-tools"
 BMO_E2E_EMULATOR=${BMO_E2E_EMULATOR:-"sushy-tools"}
 
+case "${GINKGO_FOCUS:-}" in
+  *upgrade*)
+    export DEPLOY_IRONIC="false"
+    export DEPLOY_BMO="false"
```

Comment:
I dont understand this, by default we dont deploy BMO even in upgrade test?

Reply:
The upgrade test has its own logic to install all these components, and they follow the `UPGRADE_DEPLOY_*` configs instead. The reason for that is because we use a different BMO version for upgrade test than the rest of the suite.
In the `fixtures` setup, we use two separate clusters for this purpose, so we could run all tests in the same run, but in `ci-e2e.sh`, we have only 1 cluster, so if we run upgrade test, we don't install anything in the BeforeSuite and leave these installations to the test setup.